static-responses/README.md

```diff
@@ -0,0 +1,64 @@
+# Static Responses
+
+There are situations where no backend requests are sent for an endpoint.
+Nevertheless, a response is needed. So a `response` block can exist in an
+`endpoint` block without any `proxy` or `request` blocks.
+
+Consider the following example, in which the client is instructed to perform a
+redirect via a `303` status code an a `location` header specifying the redirect
+target:
```

Comment:
`301` from `/` to `/app/` could be an example for the good practice of having the SPA paths not under `/**`.